spec/spec_helper.rb

```diff
@@ -51,6 +51,7 @@ def coverage_needed?
 require 'webmock/rspec'
 WebMock.disable_net_connect!(allow_localhost: true)
 
+require 'i18n/debug' if ENV['I18N_DEBUG']
```

Comment:
Does this introduce a weirdness if we add `ENV['I18N_DEBUG']` in Travis? I.e., should the `hyrax.gemspec` change above also check the value of `ENV['I18N_DEBUG']`, or rectify this when needed?